src/NLight/NLight.csproj

```diff
@@ -1,22 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <Version>2.1.1</Version>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Version>2.2.0</Version>
     <Copyright>Copyright (c) 2009 Sébastien Lorion</Copyright>
     <Authors>Sébastien Lorion</Authors>
     <Company />
     <Description>Toolbox for .NET projects</Description>
     <PackageProjectUrl>https://github.com/slorion/nlight</PackageProjectUrl>
     <PackageTags>io,parser,csv,delimited,transaction,reactive,tree</PackageTags>
-    <PackageReleaseNotes>Added multi-targeting support for .NET 4.6</PackageReleaseNotes>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>C:\projects\oss\nlight\src\NLight.snk</AssemblyOriginatorKeyFile>
+    <PackageReleaseNotes>Increased compatibility to newer .net versions. Removed support for .NET 4.6</PackageReleaseNotes>
```

Comment:
Please state minimal supported version (.NET 6)

Reply:
With netstandard2.0 and System.Reactive.Core.5.0.0 we can keep the library backwards compatible to Framework 4.7.2.